### DISCUSSION TITLE
Demographic Breakdowns

### DISCUSSION THREAD

**Discussion by ccunningham101** [opensafely/qof-utilities#17] [General]
Which demographic subgroups should we look at? 
[SRO-template](https://github.com/opensafely/sro-template/blob/master/analysis/config.py#L9) uses age_band, sex, imd, region, learning disability, care home status, ethnicity
[SRO-measures](https://github.com/opensafely/SRO-Measures/blob/master/analysis/study_definition.py#L11) uses age_band, sex, imd, region, learning disability, ethnicity

[Pincer](https://github.com/opensafely/pincer-measures/blob/main/analysis/calculate_measures.py#L20 ) uses age_band, sex, imd, region, ethnicity

Which ethnicity codelist should we use?
For now go with [PRIMIS vaccine codes](https://www.opencodelists.org/codelist/primis-covid19-vacc-uptake/eth2001/v1/) but should Chinese be in "Other"?
Open question about 6 versus 16 (data quality for 16?), but go with 6 for now
Should we supplement with SUS like in [PINCER](https://github.com/opensafely/pincer-measures/blob/main/analysis/study_definition_ethnicity.py#L31-L38)

Assume region is nuts1_region_name

**Comment by rose-higgins** [comment]
I have written down the results of the final decisions on the demographic breakdowns [here](https://github.com/opensafely/qof-utilities/wiki/Standard-parameters-and-variables-in-QOF-register-studies)
